server/views/pages/incidentContact.njk

```diff
@@ -91,6 +92,32 @@
         name: "incidentAvailability",
         value: formValues.incidentAvailability
       }) }}
+      
+{{ govukSelect({
+  id: "subject",
```

Comment:
To make this and `name` consistent with other field names, it'd be better as `incidentSubject`